role-bindings/generic-service-user-bind.yml

```diff
@@ -0,0 +1,39 @@
+---
+kind: Template
+apiVersion: v1
+metadata:
+  name: "add-service-user-to-group"
+  annotations:
+    openshift.io/display-name: Group and role binding Access template
+    description: RoleBinding to give access to project
+    iconClass: icon-jenkins
+labels:
+  template: role-to-group-binding
+objects:
+- apiVersion: v1
+  kind: RoleBinding
+  metadata:
+    name: serviceaccount-${PIPELINES_NAMESPACE}-${DEPLOYER_USER}-edit-role
+  roleRef:
+    name: "${ROLE}"
+  subjects:
+  - kind: ServiceAccount
+    name: "${DEPLOYER_USER}"
+    namespace: "${PIPELINES_NAMESPACE}"
+  userNames:
+  - system:serviceaccount:${PIPELINES_NAMESPACE}:${DEPLOYER_USER}
```

Comment:
This is not mandatory and can't be removed as is calculated after the values you have already added